setcp.sh

```diff
@@ -1,18 +1,20 @@
-JAR=webgraph
+#!/bin/sh
 
-sourcedir=$(cd $(dirname ${BASH_ARGV[0]}) && pwd) 
-count=$(\ls -1 $sourcedir/$JAR-*.jar 2>/dev/null | wc -l)
+sourcedir=$(cd -- "$(dirname "$0")" && pwd)
+webgraphjars=$(find "$sourcedir" -name "*.jar" | grep "webgraph-")
```

Comment:
This is problematic should I copy this script for other projects. For example, webgraph-big has webgraph in its dependencies, so you'd get a false positive for duplicate jars. Please move back to ls in sourcedir.

Reply:
You are right.
I think `-maxdepth 1` will do the trick,
else let me know and I will revert